jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/jpa/ExecutorQueryServiceImpl.java

```diff
@@ -236,9 +236,12 @@ public RequestInfo execute(Context context) {
 
                 if ("org.jbpm.persistence.jpa.hibernate.DisabledFollowOnLockOracle10gDialect".equals(ctx.get("hibernate.dialect"))) {
 
-                    Number lockedRequest = ctx.nativeQueryAndLockWithParametersInTransaction("NativePendingRequestsForProcessing", params, true, Number.class);
-                    if (lockedRequest != null) {
-                        request = ctx.find(org.jbpm.executor.entities.RequestInfo.class, lockedRequest.longValue());
+                    Object[] lockedRequest = (Object[])ctx.nativeQueryAndLockWithParametersInTransaction("NativePendingRequestsForProcessing", params, true, Object.class);
+                    if (lockedRequest != null && lockedRequest.length > 0) {
+                        try {
+                            request = ctx.find(org.jbpm.executor.entities.RequestInfo.class, ((Number)lockedRequest[0]).longValue());
+                        } catch(NumberFormatException nfe) {
+                        }
```

Comment:
Shouldn't we include there at least a warning log message? Empty catch blocks are really not a good practice even if we know that there is a very little chance of throwing that exception.